Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -34,7 +34,7 @@
 # step1 gensim: for 2018 HI prod
 step1Up2018HiProdDefaults = merge ([{'--eventcontent':'RAWSIM'},step1Up2018HiDefaults])
 # step1 gensim: for 2021 HI prod
-step1Up2021HiProdDefaults = merge ([{'--conditions':'auto:phase1_2022_realistic_hi','--era':'Run3_pp_on_PbPb','--beamspot':'Run3RoundOptics25ns13TeVLowSigmaZ','--eventcontent':'RAWSIM','--geometry':'DB:Extended'},step1Up2018HiDefaults])
+step1Up2021HiProdDefaults = merge ([{'--conditions':'auto:phase1_2022_realistic_hi','--era':'Run3_pp_on_PbPb','--beamspot':'Realistic25ns13p6TeVEarly2022Collision','--eventcontent':'RAWSIM','--geometry':'DB:Extended'},step1Up2018HiDefaults])
```

Comment:
This looks like a mistake. 
In the master version of this PR: https://github.com/cms-sw/cmssw/pull/38760 the >= 2022 scenarios got updated because the corresponding PR updating the PU libraries was changed the later Run-3 years as well (see https://github.com/cms-sw/cmssw/pull/39041) but in this case this is not appropriate. 
I suggest this bug-fix PR https://github.com/cms-sw/cmssw/pull/40238 to resolve the issue. 
Alternatively we'd need updates to GlobalTags as well as PU libraries.